desktopexporter/internal/app/components/sidebar-view/sidebar.tsx

```diff
@@ -11,14 +11,14 @@ const sidebarCollapsedWidth = 70;
 type SidebarProps = {
   isFullWidth: boolean;
   toggleSidebarWidth: () => void;
-  traceSummaries: TraceSummaryWithUIData[];
+  traceSummaries: Map<string, TraceSummaryWithUIData>;
```

Comment:
I know `Map` preserves insertion order, but it feels wrong somehow to rely on this behavior vs a sorted array